tests/test_batch_change.py

```diff
@@ -117,3 +132,75 @@ def test_list_batch_change_summaries(mocked_responses, vinyldns_client):
         assert l.status == r.status
         assert l.id == r.id
         assert l.owner_group_id == r.owner_group_id
+
+
+def test_approve_batch_change(mocked_responses, vinyldns_client):
+    arc = AddRecordChange(forward_zone.id, forward_zone.name, 'foo', 'foo.bar.com', RecordType.A, 200,
+                          AData('1.2.3.4'), 'Complete', 'id1', 'system-message', 'rchangeid1', 'rsid1')
+
+    drc = DeleteRecordSetChange(forward_zone.id, forward_zone.name, 'baz', 'baz.bar.com', RecordType.A, 'Complete',
+                                'id2', 'system-message', 'rchangeid2', 'rsid2')
+
+    bc = BatchChange('user-id', 'user-name', datetime.utcnow(), [arc, drc],
+                     'bcid', 'Complete', 'ManuallyApproved',
+                     comments='batch change test', owner_group_id='owner-group-id',
+                     reviewer_id='admin-id', reviewer_username='admin',
+                     review_comment='looks good', review_timestamp=datetime.utcnow())
+
+    approval = ApproveBatchChangeRequest(review_comment='looks good')
+
+    mocked_responses.add(
+        responses.POST, 'http://test.com/zones/batchrecordchanges/bcid/approve',
+        body=to_json_string(bc), status=200
+    )
+
+    r = vinyldns_client.approve_batch_change('bcid', approval)
+
+    assert r.user_id == bc.user_id
```

Comment:
Can we create a shared method that runs these assertions similar to `check_single_changes_are_same`?